spec/controllers/api/collectivities_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+describe Api::CollectivitiesController, type: :controller do
+  describe "GET index" do
+    subject(:body) do
+      get :index
+      response.parsed_body
+    end
+
+    let(:collectivities) do
+      (1..5).to_a.reverse.map do |i|
```

Comment:
```suggestion
      (1..2).to_a.map do |i|
```

1. en créer 5 n'est clairement pas important, >1 suffit
2. le reverse n'apporte rien à la lumière de tes tests